applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1770 @@
+{
```

Comment:
`plt.show()` may not be necessary but I added it because on the spectral analysis notebook @cyschneck requested it. Maybe we should discuss a style guide here?

The semicolon suppresses the output of the line of text (figure object information) that appears before the plot. I think it looks cleaner without displaying that line.